nav2_theta_star_planner/include/nav2_theta_star_planner/theta_star.hpp

```diff
@@ -0,0 +1,306 @@
+// Copyright 2020 Anshumaan Singh
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NAV2_THETA_STAR_PLANNER__THETA_STAR_HPP_
+#define NAV2_THETA_STAR_PLANNER__THETA_STAR_HPP_
+
+#include <cmath>
+#include <chrono>
+#include <vector>
+#include <queue>
+#include <algorithm>
+#include "rclcpp/rclcpp.hpp"
+#include "nav2_costmap_2d/costmap_2d_ros.hpp"
+
+const double INF_COST = DBL_MAX;
+const int LETHAL_COST = 250;
```

Comment:
Should this go back to 252?

Reply:
Uhh i didnt do that cause it would have to re do the tuning work to get the params. That would also mean changing the function `getCost()` to 26 + 0.9*<actual_costmap_cost>

Reply:
That would be OK with me. 251 is still valid. While functionally I don't think this will make nearly _any_ impact. Someone later is going to find this and submit a bug ticket about it. I'd rather handle it here and now